tests/virt/node/workload_density/test_swap.py

```diff
@@ -68,9 +70,9 @@ def wasp_agent_daemonset():
 
 
 @pytest.fixture(scope="package")
-def skip_if_wasp_agent_disabled(wasp_agent_daemonset):
+def xfail_if_wasp_agent_disabled(wasp_agent_daemonset):
     if not wasp_agent_daemonset.exists:
-        pytest.skip("Wasp agent not deployed to cluster, skipping test")
+        pytest.xfail(reason="Wasp agent not deployed to cluster")
```

Comment:
since you exclude swap from t2 and adding dedicated marker, i think this should be hard `fail` and not `xfail`

Reply:
good point